sdks/python/apache_beam/dataframe/frames.py

```diff
@@ -3392,7 +3395,7 @@ def melt(self, ignore_index, **kwargs):
 
   @frame_base.with_docs_from(pd.DataFrame)
   def value_counts(self, subset=None, sort=False, normalize=False,
-                   ascending=False):
+                   ascending=False, dropna=True):
```

Comment:
There was a note about "dropna=False is new in pandas 1.3" below. Should we match that?

Reply:
We don't have to, we reuse the pandas docs using the above annotation which already includes this notice.

Reply:
I was not asking about the notice. I was asking should we change dropna=True to dropna=False here in value_counts?

Reply:
Ah I understand now. We keep dropna=True here to keep backwards compatibility with Pandas version < 1.3